Improve Python Bindings buildings doc (#3215)
f3d-app/f3d: doc/dev/05-BUILD.md

> 
> `cmake --preset=vcpkg /path/to/source`
> 
> ## Python bindings
> 
> To build the Python bindings, create and activate a Python virtual environment first:
> 
>     python -m venv .venv
>     source .venv/bin/activate
> 
> Install the required Python dependencies:
> 
>     pip install --group dev
> 
> > **Note**: Require pip 25.1 or higher. You can upgrade pip using `python -m pip install --upgrade pip`.
> 
> Configure CMake with the `F3D_BINDINGS_PYTHON` option enabled:
> 
>     cmake -DF3D_BINDINGS_PYTHON=ON [...]
> 
> Build the project:
> 
>     cmake --build . --target pyf3d
> 
> Run the Python bindings tests with ctest:
> 
>     ctest -L python

Reviewer: Soemthing like that? Waht do you think @mwestphal ?

```suggestion
## Python bindings

### Requirements

Only `pybind11` should be available on your system to build python bindings. Additionally, `pytest` and `numpy` must be available to run tests.

In case of issues getting these dependencies, you can optionally create a virtual environment:

    python -m venv .venv
    source .venv/bin/activate

Install the required Python dependencies:

    pip install --group dev

> **Note**: Require pip 25.1 or higher. You can upgrade pip using `python -m pip install --upgrade pip`.

### Build

Configure CMake with the `F3D_BINDINGS_PYTHON` option enabled:

    cmake -DF3D_BINDINGS_PYTHON=ON [...]

Build the project:

    cmake --build . --target pyf3d

### Test

Run the Python bindings tests with ctest:

    ctest -L python
```

Reviewer 2: that would work for me.

`cmake --preset=vcpkg /path/to/source`

## Python bindings

### Requirements

Only `pybind11` should be available on your system to build python bindings. Additionally, `pytest` and `numpy` must be available to run tests.

In case of issues getting these dependencies, you can optionally create a virtual environment:

    python -m venv .venv
    source .venv/bin/activate

Install the required Python dependencies:

>     pip install --group dev

Reviewer 3: ```suggestion
`pybind11` is needed to build the Python bindings. Additionally, `pytest` and `numpy` must be available to run the tests. These dependencies can be installed with `pip`. It is recommended (but not strictly necessary) to install in a virtual environment.

The virtual environment can be created using the `venv` module:

    python -m venv .venv
    source .venv/bin/activate

Using `pip` to install the required Python dependencies:

    pip install --group dev

```

Or is `pybind` needed both on the system and in the venv?

Also github won't let me have it as a suggestion but the rest of the document uses triple backticks for code blocks (instead of indentation)

Reviewer 3: So what's the deal with `pybind`? The text still says it's needed on the system but the `dev` group installs it in the venv too, do we need both?

> **Note**: Require pip 25.1 or higher. You can upgrade pip using `python -m pip install --upgrade pip`.

### Build

> Configure CMake with the `F3D_BINDINGS_PYTHON` option enabled:

Reviewer 2: ```suggestion
Configure F3D with the `F3D_BINDINGS_PYTHON` CMake option enabled:
```
